events/router.js

```diff
@@ -0,0 +1,115 @@
+const { Router } = require('express');
+const eventRouter = new Router();
+const Event = require('./model');
+const auth = require('../auth/middleware');
+const { Op } = require('sequelize');
+
+
+// Create a new event
+eventRouter.post(
+  '/events', auth,
+  async (request, response, next) => {
+    try {
+      const Id = request.user.id
+      const authEvent = { ...request.body.newEventData, userId: Id }
+      const event = await Event.create(authEvent)
+      response.json(event)
+
+    } catch (error) {
+      next(error)
+    }
+  }
+)
+
+// Read future events
+eventRouter.get(
+  '/events',
+  async (request, response, next) => {
+    try {
+      const today = new Date();
+      const day = String(today.getDate() - 1).padStart(2, '0');
+      const month = String(today.getMonth() + 1).padStart(2, '0')
```

Comment:
check out moment.js for anything date related, great for manipulating, adding, substracting and then displaying in many ways https://momentjs.com/docs/#/displaying/

Reply:
Great thanks!